cacti/test_cacti.py

```diff
@@ -0,0 +1,126 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import logging
+import os
+import shutil
+from nose.plugins.attrib import attr
+
+# project
+from shared.test.common import AgentCheckTest, Fixtures
+
+log = logging.getLogger()
+
+CONFIG = """
+init_config:
+
+instances:
+    -   mysql_host: localhost
+        mysql_user: root
+        rrd_path:   /tmp/cacti_test/rrds
+        rrd_whitelist: %s
+""" % Fixtures.file('whitelist.txt')
+
+@attr(requires='cacti')
+class TestCacti(AgentCheckTest):
+
+    CHECK_NAME = 'cacti'
+
+    CACTI_CONFIG = {
+        'mysql_host': 'localhost',
+        'mysql_user': 'root',
+        'rrd_path':   '/tmp/cacti_test/rrds',
+        'rrd_whitelist': "%s" % Fixtures.file('whitelist.txt')
+    }
+
+    def setUp(self):
+        self.tmp_dir = '/tmp/cacti_test'
+        self.rrd_dir = os.path.join(os.path.dirname(__file__), "cacti")
```

Comment:
I'm not sure we have anything in this path... that would be the path where the test is so `SDK_ROOT/cacti/cacti`, I believe that doesn't exist.